### PR TITLE
DOC-1962: The image would not be inserted when the quickimage button was used on Chrome.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1962: The image would not be inserted when the quickimage button was used on Chrome.

Changes:
* added `DOC-1962: The image would not be inserted when the quickimage button was used on Chrome.` to staging.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 1]
```diff
@@ -76,6 +76,15 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== The image would not be inserted when the quickimage button was used on Chrome.
```
Comment (vpyshnenko) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:79]:
```suggestion
=== The image would not be inserted when the `quickimage` button was used on Chrome.
```

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 2]
```diff
@@ -76,6 +76,15 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
+
+In previous versions of {productname}, on MacOS Chrome, the editor `focusin` event would occur before the change event for a file input.
```
Comment (vpyshnenko) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:81]:
```suggestion
In previous versions of {productname}, on MacOS Chrome, the editor `focusin` event would occur before the `change` event for a file input.
```

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 3]
```diff
@@ -76,6 +76,15 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
+
+As a consequence, images could not be inserted via `quickimage` button of quickbar plugin.
```
Comment (vpyshnenko) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:83]:
```suggestion
As a consequence, images could not be inserted via `quickimage` button of `quickbar` plugin.
```

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 4]
```diff
@@ -76,6 +76,15 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
...
+
+To fix this, the handling of `focusin` events was delayed in order to give a time slot for the change event to be handled.
```
Comment (vpyshnenko) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:85]:
```suggestion
To fix this, the handling of `focusin` events was delayed in order to give a time slot for the `change` event to be handled.
```

**Review by vpyshnenko — APPROVED** [review]

File: modules/ROOT/pages/6.4.2-release-notes.adoc [hunk 5]
```diff
@@ -76,6 +76,13 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== The `quickimage` toolbar button failed to insert images selected from the local computer when running on Google Chrome for macOS 
+
+A regression was discovered in {productname} 6.4.0 when the macOS version of Google Chrome was the host browser: the `focusin` editor event occurred before the `change` event for a file input.
+
+As a consequence, although selecting the `quickimage` toolbar button presented the macOS-native *NSOpenPanel* dialogue box as expected, images selected from this dialogue box were not inserted into the {productname} document.
+
+For this upate, the regression has been correct. In {productname} 6.4.2, local images selected after the `quickimage` button is selected are added to the {productname} document, as expected.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.4.2-release-notes.adoc:85]:
```suggestion
For this upate, the regression has been corrected. In {productname} 6.4.2, local images selected after the `quickimage` button is selected are added to the {productname} document, as expected.
```

Typo correction.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**